extra-multimedia/kodi/autobuild/prepare

```diff
@@ -0,0 +1,10 @@
+RESDIR="$SRCDIR/../repo-resources-4a733e3f10ee41f0d639376b11706d916974ae94"
```

Comment:
If Git source is used for Kodi resources, you would not have needed to reference a hard-coded Git commit hash here.